lib/geocoder/agolgeocoder.js

```diff
@@ -119,7 +119,6 @@ AGOLGeocoder.prototype.geocode = function(value, callback) {
     };
 
     _this.httpAdapter.get(_this._endpoint, params, function(err, result) {
-      result = JSON.parse(result);
```

Comment:
If you would like to remain backwards compatible you could leave this in with a test on `result` to see if it's an object or a string.

It seems like the endpoint response has changed since the original addition some years back.